packages/@react-spectrum/card/test/CardView.test.js

```diff
@@ -1190,6 +1190,7 @@ describe('CardView', function () {
       fireEvent.keyUp(document.activeElement, {key: 'End', code: 35, charCode: 35});
       act(() => {
         jest.runAllTimers();
+        jest.runAllTimers(); // added after empty Virtualizer focus change
```

Comment:
just curious, is this change still needed?

Reply:
It doesn't work. :(